Code/Tools/AssetProcessor/native/utilities/ApplicationManagerBase.h

```diff
@@ -184,9 +184,9 @@ protected Q_SLOTS:
 
 protected:
     int m_processedAssetCount = 0;
-    int m_failedAssetsCount = 0;
     int m_warningCount = 0;
     int m_errorCount = 0;
+    AZStd::unordered_set<AZStd::string> m_failedAssets;
```

Comment:
If we use a regular set here, then this would have the failed assets be output in lexical order